navigation.md

```diff
@@ -0,0 +1,19 @@
+# Learn Braille navigation
+
+- main menu
+  - theory
+    - continue last visited course
+    - courses [Golubina, ...] `no menu while we have one course`
+      - continue
+      - lessons [lesson 1, ...]
+        - steps
+  - practice
+    - browse symbols
+    - decks [all, ru symbols, punctuation, ...], 
```

Comment:
decks: **not right now** because it's unclear where to switch them on and off (remember, for the sake of accessibility there should be MINIMUM actions required to use the app so I don't want to add one more menu between main menu and practice)
 but I like this idea, probably we shall add it in a sidebar menu

Reply:
I've added bold font for items, which concept is not proved.

Reply:
But yes, we will have one more menu, but the very first button on this menu will be `All cards`. So user is supposed only to quickly select it if he wants usual practice behavior. Maybe it is not so much, and adding some complicated navigation way will be worse.

Reply:
It should be as easy as possible to get to the course you are already on (as well as to get to the deck in practice you were on your previous session). So wouldn't it make sense to navigate right to the step the person stopped at (or to the deck one has previously been practicing)? All the menu options can then be easily accessed by clicking the up button.
If no course or deck are started the person has to choose first.

Reply:
anyway, we should not add this option until an overflow menu is implemented. I suppose that choosing decks should be placed there and some defaults should be preset.